app/services/editions/EditionsTemplating.scala

```diff
@@ -30,7 +30,7 @@ class EditionsTemplating(templates: PartialFunction[Edition, EditionTemplate], c
                     EditionsCollectionSkeleton(
                       collection.name,
                       collection.prefill.map { prefill =>
-                        val prefillParams = PrefillParamsAdapter(issueDate, prefill, edition)
+                        val prefillParams = PrefillParamsAdapter(issueDate, prefill, frontTemplate.ophanUrl, template.ophanRangeSearchDays, edition)
                         getPrefillArticles(prefillParams)
```

Comment:
as i mention above, will prefer to pass some `OphanQueryConfig` object here not url and range separately